annotations/views.py

```diff
@@ -167,21 +168,32 @@ def user_settings(request):
     """ User profile settings"""
 
     if request.method == 'POST':
-        form = UserChangeForm(request.POST, instance=request.user)
+        form = UserChangeForm(request.POST, request.FILES,instance=request.user)
```

Comment:
Now that we have S3 upload, we shouldn't be handling any files here. Just the URL for the image on S3, which should be in the ``POST`` data.